### ISSUE TITLE
Excess inclusions?

### ISSUE THREAD

**Issue by Riim** [rollup/rollup#975]
About version 0.35 in the bundle began to get these inclusions:

``` js
var get$1 = function get$1(object, property, receiver) {
  if (object === null) object = Function.prototype;
  var desc = Object.getOwnPropertyDescriptor(object, property);

  if (desc === undefined) {
    var parent = Object.getPrototypeOf(object);

    if (parent === null) {
      return undefined;
    } else {
      return get$1(parent, property, receiver);
    }
  } else if ("value" in desc) {
    return desc.value;
  } else {
    var getter = desc.get;

    if (getter === undefined) {
      return undefined;
    }

    return getter.call(receiver);
  }
};



var set = function set(object, property, value, receiver) {
  var desc = Object.getOwnPropertyDescriptor(object, property);

  if (desc === undefined) {
    var parent = Object.getPrototypeOf(object);

    if (parent !== null) {
      set(parent, property, value, receiver);
    }
  } else if ("value" in desc && desc.writable) {
    desc.value = value;
  } else {
    var setter = desc.set;

    if (setter !== undefined) {
      setter.call(receiver, value);
    }
  }

  return value;
};
```

This code is not used, and was not added me.

Why is it?
Thanks.
## 

for repeating:

```
> npm i rionite && cd rionite
> npm i -D rollup
> npm run build
> nano dist/Rionite.js
line 40
```

**Comment by rturnq** [comment]
I too am seeing `get` and `set` included in my bundle using v0.36.1.   It appears these extraneous bits of code are coming from `babel-helpers` when using the `external-helpers` Babel plugin and your code contains _anything_ that is transformed into a helper function.

Additionally, I am getting the whole `AsyncGenerator` class included even though I don't have any async functions in my source.

**Comment by Rich-Harris** [comment]
[Please include a repro](https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff)
